### PR TITLE
data-testid added in navbar and admin

### PR DESCRIPTION
**Pull request by Ajay-aot**
JIRA:
Issue Type: BUG/ FEATURE

https://aottech.atlassian.net/browse/FWF-2909
data-test id added in navbar and admin

### CODE REVIEW

File: forms-flow-admin/src/components/roles/roles.tsx [hunk 1]
```diff
@@ -249,7 +249,7 @@ const Roles = React.memo((props: any) => {
   // Delete confirmation
 
   const confirmDelete = () => (
-    <div>
+    <div data-testid="confirm-delete-modal">
```
Comment (shuhaib-aot) [inline forms-flow-admin/src/components/roles/roles.tsx:252]: please add the data test id with more specific like role-confirm-delete-modal

**Review by shuhaib-aot — COMMENTED** [review]
plese give the page name as prefix of data test id or create data test including the page name also

File: forms-flow-admin/src/components/roles/roles.tsx (hunk 1 above)
Comment (Ajay-aot) [reply forms-flow-admin/src/components/roles/roles.tsx]: updated

**Comment by fahad-aot** [comment]
can you please add data-testid for the search role or user and its clear button

**Review by shuhaib-aot — APPROVED** [review]

**Pull request merged**